src/context/AuthContext.tsx

```diff
@@ -60,28 +59,13 @@ export const AuthProvider: React.FC<ProviderProps> = ({ children }) => {
     const loadProject = async (user: User) => {
         const project = await handleGetProject(user.uid)
         if (project) {
-            // console.log('existing project:', project)
-            // Check if they have an old-formatted project
-            // to convert to the new user-id based project
-            // const oldProject = await (await handleSearchProjects(project.name)).find(p => p.id !== id)
-            // if (oldProject) {
-            //     console.log('found old project:', oldProject)
-                
-            //     const updatedProject: Project = {
-            //         ...oldProject,
-            //         id: project.id
-            //     }
-            //     console.log('updating new project to:', updatedProject)
-            //     await handleUpdateProject(updatedProject)
-            //     setCurrentProject(updatedProject)
-            // } else {
-            //     setCurrentProject(project)
-            // }
-            setCurrentProject(project)
+            // TODO: check if project is using old data design
```

Comment:
Need to write a temporary transition script to convert all the existing project data to the new subcollections.

Reply:
Data migrations complete.